pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -139,7 +147,9 @@ def __init__(self,
         seedValues: list of int
             Seed value for each seed of requested TurbSim simulations if nSeeds!=6
         LESpath: str or list of strings
-            Full path of the LES data, if driven by LES. If None, the setup will be for TurbSim inflow.
+            Full path of the LES data, if driven by LES. 
+            If TurbSim, the setup will be for TurbSim inflow.
+            If MannBox, the setup will be for MannBox inflow.
```

Comment:
Okay, I see a need for a different option if now we have TurbSim, Mann, and LES. This needs to be changed.